sdk/eventgrid/Microsoft.Azure.Management.EventGrid/EventGrid.sln

```diff
@@ -3,9 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27130.2020
```

Comment:
@shahabhijeet are the solutions files needed or even used for the auto-generated management libraries?